src/app/containers/PodcastExternalLinks/index.jsx

```diff
@@ -18,13 +18,16 @@ import { ServiceContext } from '#contexts/ServiceContext';
 import Link from './Link';
 
 const EN_GB_LANG = 'en-GB';
+const HINDI_LANG = 'hi';
 
 const ExternalLinkTextLangs = {
   Spotify: EN_GB_LANG,
   Apple: EN_GB_LANG,
   RSS: EN_GB_LANG,
   Yandex: EN_GB_LANG,
   Castbox: EN_GB_LANG,
+  'Jio Saavn': HINDI_LANG,
```

Comment:
This and the one below doesn't need a lang if the page they are on is Hindi already, they will just inherit it.